client/src/components/Draftail/CommentableEditor/CommentableEditor.tsx

```diff
@@ -445,6 +449,8 @@ function CommentableEditor({
     comments,
   ]);
 
+  const controls = useMemo(() => (enabled ? [CommentControl] : []), [enabled, CommentControl]);
```

Comment:
Why do we need a `useMemo` here? This seems like it'll be pretty fast

Reply:
Yep, this can be probably be removed actually. I was testing whether using an inline array here caused some of the event listener problems, but I don't think this did much!

Reply:
Feel free to merge once this is done